core/src/main/resources/AppConstants.properties

```diff
@@ -115,8 +115,12 @@ jdbc.required=true
 jdbc.datasource.default=jdbc/${instance.name.lc}
 
 ## JDBC Connection Pool properties
-# Minimum number of physical connections to maintain.
+# Deprecated, use instead `transactionmanager.jdbc.connection.minIdle`
```

Comment:
Shouldn't we remove the deprecated option from 8.1 because we already marked it as deprecated in 7.9/8.0 and people are still going to migrate to that version still?

Reply:
Regardless when people migrate to newer versions, it's deprecated in this one, thus the tag should be there?

Reply:
Ok, we'll leave it for 8.1 like this.